lib/v2/liquipedia/cs_matches.js

```diff
@@ -0,0 +1,54 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const team = ctx.params.team ?? 'G2_Esports';
```

Comment:
`team` should not have any default value.

Reply:
Thanks for review.
I mainly take this as a reference, any difference?
https://github.com/DIYgod/RSSHub/blob/068457cd277dc76efe33e3658f54f59970ac9895/lib/v2/lofter/tag.js#L6-L7

Reply:
Because `name` and `type` are decleared as optional parameter https://github.com/DIYgod/RSSHub/blob/068457cd277dc76efe33e3658f54f59970ac9895/lib/v2/lofter/router.js#L2

and `team` is decleared as required parameter
https://github.com/DIYgod/RSSHub/blob/068457cd277dc76efe33e3658f54f59970ac9895/lib/v2/liquipedia/router.js#L2

[Related docs](https://docs.rsshub.app/joinus/new-rss/before-start#routerjs)

Reply:
Thanks for your explanation. The defaut value has been removed.